src/vm/eventpipe.cpp

```diff
@@ -716,11 +716,14 @@ bool EventPipe::WalkManagedStackForThread(Thread *pThread, StackContents &stackC
 
     stackContents.Reset();
 
+    bool gcOnTransitions = GC_ON_TRANSITIONS(FALSE);                // dont do GC for GCStress 3
```

Comment:
You'll probably run in to compiler errors on non-windows platforms due to `bool`/`BOOL` since `GC_ON_TRANSITIONS` returns `BOOL`. Clang doesn't like the implicit conversion from `BOOL` to `bool`.

Reply:
Oh. I wrote/built/tested this code on my Ubuntu box but didn't run into that error. Should I change it to BOOL anyway?

Reply:
It probably doesn't matter if you don't get any warnings. I prefer to match the types exactly if possible, but I don't think there are any issues that can come out of it.